casepro/cases/views.py

```diff
@@ -341,39 +341,6 @@ def get_summary(self, partner):
                 'cases_closed': Case.objects.filter(org=partner.org, assignee=partner).exclude(closed_on=None).count()
             }
 
-    class RepliesChart(OrgObjPermsMixin, SmartReadView):
```

Comment:
Moves to `statistics/views`. Can make chart views more re-usable when they're not tied to a CRUDL